mllib/src/main/scala/org/apache/spark/mllib/clustering/DpMeansModel.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.mllib.clustering
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+
+import org.apache.spark.mllib.linalg.Vector
+import org.apache.spark.rdd.RDD
+
+/**
+ * A clustering model for DP means. Each point belongs to the cluster with the closest center.
+ */
+class DpMeansModel
+    (val clusterCenters: Array[Vector]) extends Serializable {
```

Comment:
if this line doesn't fit into L29, we should move `(` to L29:

~~~scala
class DpMeansModel(
    clusterCenters: Array[Vector]) extends Serializable {
  ...
}
~~~

Btw, it is also useful to save `lambda` in the model.